### PR TITLE
Don't hardcode kubectl in configure-helper

### PR DESCRIPTION
**Pull request by wojtek-t**
It's what I found when I was trying to run k8s on GCE with ubuntu master.

```release-note
NONE
```

### CODE REVIEW

**Comment by wojtek-t** [comment]
/hold

Something doesn't work after rebase...

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -2610,7 +2610,7 @@ EOF
 }
 
 function wait-till-apiserver-ready() {
-  until kubectl get nodes; do
+  until ${KUBECT} get nodes; do
```
Comment (dims) [inline cluster/gce/gci/configure-helper.sh]: missing `L` :)

Comment (wojtek-t) [reply cluster/gce/gci/configure-helper.sh]: thanks - good catch

**Comment by wojtek-t** [comment]
Trivial fix - reaaplying lgtm.

**Comment by liggitt** [comment]
is this still required with https://github.com/kubernetes/kubernetes/pull/87772 ?

File: cluster/gce/gci/configure-helper.sh [hunk 2]
```diff
@@ -2713,6 +2713,7 @@ function main() {
 
   KUBE_HOME="/home/kubernetes"
   KUBE_BIN=${KUBE_HOME}/bin
+	KUBECTL="/home/kubernetes/bin/kubectl"
```
Comment (liggitt) [inline cluster/gce/gci/configure-helper.sh]: nit: indent diff

**Comment by wojtek-t** [comment]
> is this still required with #87772 ?

good finding - it probably isn't. I didn't see that.
/hold

**Pull request closed without merging**